config/spring.rb

```diff
@@ -0,0 +1,6 @@
+%w(
+  .ruby-version
+  .rbenv-vars
+  tmp/restart.txt
+  tmp/caching-dev.txt
+).each { |path| Spring.watch(path) }
```

Comment:
We don't actually use `spring`, this is just part of the default configuration setup.